core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/runtimeModels.scala

```diff
@@ -203,13 +203,16 @@ sealed trait RuntimeConfig extends Product with Serializable {
 object RuntimeConfig {
   final case class GceConfig(
     machineType: MachineTypeName,
-    diskSize: DiskSize
+    diskSize: DiskSize,
+    bootDiskSize: Option[DiskSize]
```

Comment:
Should we maybe add a comment that this is an `Option` to support existing runtimes, and all runtimes going forward will have a bootDiskSize?